app/presenters/tree_builder_pxe_customization_templates.rb

```diff
@@ -11,8 +11,7 @@ def set_locals_for_render
   end
 
   def root_options
-    title = _("All %{template} - %{type}") % {:template => ui_lookup(:models => 'CustomizationTemplate'),
-                                              :type     => ui_lookup(:models => 'PxeImageType')}
+    title = _("All Customization Templates - System Image Types")
```

Comment:
True, but the real reason for doing this change is translations: having model name in one place and then inserting it into other strings makes sense in English (and looks just fine in English), but doesn't
really work in other locales that have more complex conjugation rules.